tests/build.snapshot.js

```diff
@@ -7,7 +7,8 @@ const smallBuildCommons = {
 	'polyfills.js': { size: 4620 },
 	'polyfills.js.map': { size: 31760 },
 	'favicon.ico': { size: 15086 },
-	'sw.js': { size: 3330 },
+	'sw.js': { size: 1628 },
+	'workbox-sw.prod.v1.1.0.js': {size: 47026},
```

Comment:
@lukeed I remember your only concern about `offline-plugin` was its size so I wasn't really suggesting a change. But hey, if you're going to migrate to `workbox` then `offline-plugin` could be a candidate for migration too.